packages/backend/src/scheduler/SchedulerClient.ts

```diff
@@ -137,35 +138,69 @@ export class SchedulerClient {
         return { jobId: id, date };
     }
 
+    private async addGsheetsNotificationJob(
+        date: Date,
+        jobGroup: string,
+        scheduler: Scheduler,
+    ) {
+        const graphileClient = await this.graphileUtils;
+
+        const payload: GsheetsNotificationPayload = {
+            schedulerUuid: scheduler.schedulerUuid,
+            jobGroup,
+            scheduledTime: date,
+        };
+
+        const { id } = await graphileClient.addJob(
```

Comment:
Nice and clean!! Payload makes sense. Naming is just a bit confusing if I was looking at the job name in analytics or in the UI

Reply:
I updated gsheet task name